internal/sail/server/server.go

```diff
@@ -43,14 +52,25 @@ func (s SailServer) newRoom(conn *websocket.Conn) *Room {
 
 	room := NewRoom(conn)
 	s.rooms[room.id] = room
+	log.Printf("newRoom: %s", room.id)
```

Comment:
👀

Reply:
There are a bunch of these logs: are they meant to be used long term? If so can we make them structured logs so they'd be easy to pick up in Honeycomb?

Reply:
yep, the logs are intentional. wanted to get something working and debuggable before we start thinking about logging strategy. a lot will need to change before we can even deploy this